README.md

```diff
@@ -27,89 +27,89 @@ Last update: 2021-05-14
 
 |     | Checkov | Indeni Cloudrail | Kics | Snyk | Terrascan | Tfsec |
 |----|----|----|----|----|----|----|
-|Tested Version|2.0.135|1.2.130|1.3.0|1.563.0|1.4.0|0.39.34|
-|Total Catch Rate|66%|69%|31%|43%|16%|53%|
+|Tested Version|2.0.135|1.2.130|snapshot-554d322c|1.563.0|1.4.0|0.39.34|
```

Comment:
@rogeriopeixotocx is this an officially available version of KICS? Seems like a dev build.

Reply:
This image is the result of cloning the KICS git repository and running `make docker`.

Reply:
Users don't do that. They pull a GA-supported docker. KICS's official documentation tells users to use `:lastest`: https://hub.docker.com/r/checkmarx/kics

And so the testing here needs to use latest.